SU-SOE/engineering_profile
# Soed9 21

# READY FOR REVIEW

# Summary
- This PR brings us up to date with the configs, updates, and tests from `soe_profile` from the 8.x-2.x branch, current as of 06/11/2020, so it includes all the stuff going out in the 2.1.2 release of ACSF stacks.  This also is the first branch that passes tests on circleci.

# Needed By (Date)
- ASAP.

# Urgency
- Sets us a new baseline.

# Steps to Test

1. Check out this branch.
2. drush cim
3. Make sure there are no problems.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
